docs_user/modules/proc_configuring-openshift-worker-nodes.adoc

```diff
@@ -1,18 +1,16 @@
 [id="configuring-openshift-worker-nodes_{context}"]
 
-= Configuring {OpenShiftShort} worker nodes
+= Configuring isolated networks on {OpenShiftShort} worker nodes
 
-{rhocp_long} worker nodes that run {rhos_prev_long} services need a way to connect the service
-pods to isolated networks. This requires physical network configuration on the
-hypervisor.
+To connect service pods to isolated networks on {rhocp_long} worker nodes that run {rhos_prev_long} services physical network configuration on the hypervisor is required.
```

Comment:
Is it correct say "...on OpenShift worker nodes that run **OpenShift** services...."?

Reply:
No. It's saying that the new control plane services need a way to talk to services running on the RHOSP ("to-be-adopted") cluster. For this, physical network configuration for OpenShift nodes should be modified to build a path between OpenShift pods (that run the new control plane) and the isolated networks (through which they will talk to data plane node services).